### DISCUSSION TITLE
Updating Program via PowerShell?

### DISCUSSION THREAD

**Discussion by LookingForWayOut** [valinet/ExplorerPatcher#3411] [Q&A]
Hello.

Is there a way to update the program (including those symbol messages) via Windows 11 Enterprise Powershell, instead of the ExplorerPatcher's properties option?

Thank you.

**Comment by pyrates999** [comment]
This might help ya: https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To

The symbols are downloaded by EP itself after it is installed.
